ci/infra/testrunner/utils/constants.py

```diff
@@ -151,7 +151,7 @@ def finalize(conf):
         conf.workspace = os.path.expanduser(conf.workspace)
 
         if not conf.log_dir:
-            conf.log_dir = os.path.join(conf.workspace, 'testrunner_logs')
+            conf.log_dir = os.path.join(conf.workspace, 'platform_logs')
```

Comment:
I admit I don't fully understand the background of the story that led to this change but this feels slightly odd to change the path of the logdir of a python module stored inside 'testrunner' directory to default to 'platform'. 

should the whole infra/testrunner directory be renamed to platform as well?

Reply:
@pablochacin touched on this in the linked issue (SUSE/avant-garde#714); the intended purpose is to differentiate the output of test execution from the output of testrunner itself.

Reply:
> should the whole infra/testrunner directory be renamed to platform as well?

I don't think so. My understanding is that the platform logs here are like the service logs we would have collected with supportconfig in mkcloud, so calling these logs `testrunner_logs` is like calling the supportconfig tarballs `tempest_logs`, it doesn't make sense because the test runner has its own logs for the tests themselves (`tempest.log` and `testrunner.log`).

Reply:
Thanks!